### PR TITLE
Moving the %%bigquery magic to the first line of code blocks.

### PR DESCRIPTION
**Pull request by danielsparing**
When using magics like '%%biquery', this has to be on the first line, else it fails with "syntax error at create or update model". So moving the comment lines that were above it, to under it.

### CODE REVIEW

**Comment by ryangillard** [comment]
<p>LGTM</p>

File: courses/machine_learning/deepdive2/time_series_prediction/solutions/3_modeling_bqml.ipynb [hunk 1]
```diff
@@ -510,8 +510,8 @@
     }
```
Comment (BenoitDherin) [inline courses/machine_learning/deepdive2/time_series_prediction/solutions/3_modeling_bqml.ipynb]: <p>I think the "# TODO 1a" will produce an error. Better use "--" for comments in a BQ cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (BenoitDherin) [inline courses/machine_learning/deepdive2/time_series_prediction/solutions/3_modeling_bqml.ipynb]: <p>I think the "# TODO 1b" will produce an error. Better use "--" for comments in a BQ cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (BenoitDherin) [inline courses/machine_learning/deepdive2/time_series_prediction/solutions/3_modeling_bqml.ipynb]: <p>I think the "# TODO 2a" will produce an error. Better use "--" for comments in a BQ cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Comment by ryangillard** [comment]
It was the solution so I ended up removing those TODO comments anyways.
Potential problem solved.

Ryan Gillard
Machine Learning Solutions Engineer
650-203-8205

**Comment by danielsparing** [comment]
Agree that removing all is the best.
However just fyi, the BQ magic code box does tolerate # comments because
that's how it receives the dialect switch.

**Pull request merged**